ansible_base/rbac/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.urls import include, path
+
+from ansible_base.rbac.api.router import router
+from ansible_base.rbac.apps import AnsibleRBACConfig
+
+app_name = AnsibleRBACConfig.label
+
+api_version_urls = [
```

Comment:
At first I had no idea what was going on with this, but now that I do I'm unhappy about it.

Why are we doing this?  https://github.com/ansible/django-ansible-base/blob/devel/ansible_base/lib/dynamic_config/dynamic_urls.py

As far as I can tell, the only benefit is that it avoids writing a urls file.  And that is IMO a dubious benefit, since the project can't then opt-out or customize what is going on.

We seem to have 3 different flavors of url that dynamic_urls.py is processing, but as far as I can tell it isn't actually **doing** anything with that.  Why?

As far as I can tell, there's no reason why DAB couldn't either:

a) require projects to create a urls.py file that explicitly integrates only the urls for the apps that it is using, or
b) have DAB expose a top-level urls.py file in the normal Django style that conditionally does an `include()` for each module's urls.py if it is present in the settings

Reply:
This is pretty standard from https://github.com/ansible/django-ansible-base/blob/devel/docs/apps_or_lib.md

This RBAC app only adds "version" URLs, because it just adds paths like `/api/v2/role_definitions/`

Reply:
My complaint is only about 10% about the non-standard module attributes we are using to accumulate the urls, and more like 90% about the weird and brittle machinery in https://github.com/ansible/django-ansible-base/blob/devel/ansible_base/lib/dynamic_config/dynamic_urls.py